crates/egui_kittest/src/builder.rs

```diff
@@ -19,6 +21,8 @@ impl<State> Default for HarnessBuilder<State> {
             pixels_per_point: 1.0,
             state: PhantomData,
             renderer: Box::new(LazyRenderer::default()),
+            max_steps: 6,
+            step_dt: 1.0 / 6.0,
```

Comment:
Better to use a value that can be represented exactly by an `f32` imho (I'm not sure `6.0 * (1.0 / 6.0) == 1.0`, and if it is, it's by luck)

```suggestion
            max_steps: 4,
            step_dt: 1.0 / 4.0,
```